<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wavSampleClass</class>
 <widget class="QWidget" name="wavSampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>wavSample</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="labNum">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>001.  </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labTime">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;1:58&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labFileName">
     <property name="text">
      <string>40M RU6LWZ </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chbEna">
     <property name="maximumSize">
      <size>
       <width>13</width>
       <height>13</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox
{
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QDateTimeEdit" name="dteInfo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>13</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>13</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{background-color: transparent; color: rgb(128, 128, 128);}</string>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labFreq">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{background-color: transparent; color: rgb(128, 128, 128);}</string>
       </property>
       <property name="text">
        <string>7.000.000</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
